packages/CollabCodeEdit-Tests.package/CCEDeployAgentClassTest.class/instance/testBreakdown.st

```diff
@@ -0,0 +1,9 @@
+testing
+testBreakdown
+	CCEDeployAgent
+		breakdown;
+		start.
+	CCEDeployAgent currentInstance ifNil: [ self fail ].
```

Comment:
Why did you break the cascade here?

Reply:
As we send a message to an Object returned from `currentInstance`, further cascading isn't possible, is it?